src/main/kotlin/probation/model.kt

```diff
@@ -72,9 +77,10 @@ fun probationModel(model: Model) {
   interventionServices.uses(interventionsManager, "creates new interventions in")
   interventionsManager.uses(delius, "pushes contact information of interest to")
   ndmis.uses(delius, "extracts and transforms data from")
-  nomis.uses(delius, "offender data is copied into", "NDH")
-  nomis.uses(oasys, "offender data is coped into", "NDH")
-  nomis.uses(prisonToProbation, "notifies changes")
+  ndmis.uses(OffenderManagementInCustody.allocationManager, "sends extracts containing service user allocation to", "email")
+  NOMIS.system.uses(delius, "offender data is copied into", "NDH")
+  NOMIS.system.uses(oasys, "offender data is coped into", "NDH")
+  NOMIS.system.uses(prisonToProbation, "notifies changes")
```

Comment:
Hi @sldblog, could these live in `NOMIS.defineRelationships()`? If we're encapsulating that in the `NOMIS` class I think it makes sense to define it's outgoing relationships there.

Reply:
I don't know 😅

The first two systems are probation-specific, which wouldn't exist in `prison/workspace` paths. The last one is a bridging system, so probably worth having it in both.

Reply:
It looks like after #35, yes